Jenkinsfile

```
@@ -23,6 +23,10 @@ pipeline {
                             "-Ddocker.push.password=\"\${DOCKER_PUSH_PWD}\" " +
                             "-Ddocker.work.dir=\"/tmp/\${EXECUTOR_NUMBER}\" " +
                             "-Ddocker.build.tag=\"\${DOCKER_TAG_TO_USE}\" " +
+                            "-Ddocker.build.image.arg.testArg=\"true\" " +
```

this example is not consistent with the implementation
```suggestion
                            "-Ddocker.image.build.arg.testArg=\"true\" " +
```